src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/05-services.e2e-spec.ts

```diff
@@ -0,0 +1,32 @@
+import { ServicesPageHelper } from '../cluster/services.po';
+
+describe('Services page', () => {
+  const services = new ServicesPageHelper();
+
+  beforeEach(() => {
+    cy.login();
+    Cypress.Cookies.preserveOnce('token');
+    services.navigateTo();
+  });
+
+  describe('when Orchestrator is available', () => {
+    // beforeEach(function () {
+    //   cy.fixture('orchestrator/inventory.json').as('hosts');
+    //   cy.fixture('orchestrator/services.json').as('services');
+    // });
+
+    it('should create a RGW service', function () {
```

Comment:
To be aligned with others:
```suggestion
    it('should create a RGW service', () => {
```